### PR TITLE
Update presence.js

### PR DESCRIPTION
**Pull request by samet99**
Fix

### CODE REVIEW

**Comment by samet99** [comment]
Fix

File: discordtrlist.xyz/dist/presence.js [hunk 1]
```diff
@@ -6,42 +6,103 @@ var presence = new Presence({
     pause: "presence.playback.paused"
 });
 var browsingStamp = Math.floor(Date.now() / 1000);
-
 presence.on("UpdateData", () => {
     let presenceData = {
-        largeImageKey: "logo"
+        largeImageKey: "logo",
```
Comment (Bas950) [inline discordtrlist.xyz/dist/presence.js]: remove comma at the end

**Comment by Bas950** [comment]
Also upload proof that it works

File: discordtrlist.xyz/dist/presence.js (hunk 1 above)
Comment (samet99) [reply discordtrlist.xyz/dist/presence.js]: Fix

**Pull request merged**